Automattic/wp-calypso
# Enable storybook controls

Related to p7H4VZ-4gp-p2#comment-11252

## Proposed Changes

In this PR, I propose to install storybook controls addon.

## Testing Instructions



1. Run storybook in root
```
yarn storybook:start
```
3. Confirm controls addon works fine

## Pre-merge Checklist
